Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1231,9 +1231,9 @@ def fitToViewport(self, Actor, vp, wc=None, geo=None, priority=None,
                 NGridCover = 0
                 pt.SetNumberOfPoints(Npts * Npts)
                 for x in numpy.arange(
-                        Xrg[0], Xrg[1], (Xrg[1] - Xrg[0]) / Npts):
+                        Xrg[0], Xrg[1] + (Xrg[1] - Xrg[0]) / Npts, (Xrg[1] - Xrg[0]) / Npts):
```

Comment:
Here we generate a 2D grid of points to cover the whole domain and transform those points through the geographic projection. The goal is to get the bounds of the projection. The problem is that arrange uses an interval open at the higher end, so the points on the margin are not included. What I have done is to include the points on the margin.

Of course a better fix would be to pass the bounds of the projection but that will take more time to learn the code.